queries/date.py

```diff
@@ -863,9 +790,16 @@ def days_in_month_answ(q: Query, result):
     ndays = result["days_in_month"]
     t = result["target"]
     mname = t.strftime("%B")
-    answer = "{0} dagar.".format(ndays)
+    answer = f"{ndays} dagar." if is_plural(ndays) else f"{ndays} dagur."
```

Comment:
Might want to look at extending the sing_or_plur function in __init__.py, have it support your num_to_text stuff and use it for cases such as these.